security/pkg/stsservice/tokenmanager/google/tokenexchangeplugin.go

```diff
@@ -135,7 +167,18 @@ func (p *Plugin) constructFederatedTokenRequest(parameters stsservice.StsRequest
 	req, _ := http.NewRequest("POST", federatedTokenEndpoint, bytes.NewBuffer(jsonQuery))
 	req.Header.Set("Content-Type", contentType)
 	if pluginLog.DebugEnabled() {
-		reqDump, _ := httputil.DumpRequest(req, false)
+		dQuery := map[string]string{
+			"audience":           aud,
+			"grantType":          parameters.GrantType,
+			"requestedTokenType": tokenType,
+			"subjectTokenType":   parameters.SubjectTokenType,
+			"subjectToken":       "redacted",
+			"scope":              reqScope,
+		}
+		dJSONQuery, _ := json.Marshal(dQuery)
+		dReq, _ := http.NewRequest("POST", federatedTokenEndpoint, bytes.NewBuffer(dJSONQuery))
```

Comment:
why ignore all the errors here?

Reply:
This part is to dump the request for troubleshooting purposes, dJSONQuery and dReq are not sent to the endpoint.